pandas/tests/io/test_fsspec.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import pytest
+
+from pandas import DataFrame, date_range, read_csv, read_parquet
+import pandas._testing as tm
+from pandas.util import _test_decorators as td
+
+from pandas.io.common import is_fsspec_url
+
```

Comment:
everything in this module is safe if fsspec is not available?

Reply:
ok to simply skip the module

Reply:
you mean `fsspec = pytest.importorskip("fsspec")`? 
`test_is_fsspec_url` should be moved to test_common in that case.